src/Common/SelectInterface.php

```diff
@@ -232,4 +232,15 @@ public function union();
      *
      */
     public function unionAll();
+
+    /**
+     *
+     * Will select max value from a specified column
+     *
+     * @param string $col name of the Column
+     *
+     * @return self
+     *
+     */
+    public function max($col);
```

Comment:
The functionality was implemented in Common/Select. All select classes inherit from that correct? Where is the BC break?